src/core/resolvers/idux.ts

```diff
@@ -27,6 +27,12 @@ export interface IduxResolverOptions {
    * import style along with components
    */
   importStyle?: 'css' | 'less'
+  /**
+   * theme for import style
+   * 
+   * @default 'default'
+   */
+  importStyleTheme?: 'default' | 'seer'
```

Comment:
```suggestion
  importStyleTheme?: string
```

I think it shouldn't be us to maintain the theme list